script.js

```diff
@@ -1,4 +1,216 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+var scissors = "scissors";
+var stone = "stone";
+var paper = "paper";
+//Keeping track of Computer's and User's wins
+var computerWin = 0;
+var userWin = 0;
+//Keeping track of game mode
+var gameMode = "normal";
+//username
+var username = "";
+
+var main = function (inputSign) {
+  //computer's sign
+  var resultOfComputerSign = computerSign();
+  //Accounting for Gamemode Change
+  if (gameMode == "normal" && inputSign == "reverse") {
+    gameMode = "reverse";
+    var myOutputValue = "Gamemode is now in reverse";
+    return myOutputValue;
+  }
+  if (gameMode == "reverse" && inputSign == "normal") {
+    gameMode = "normal";
+    var myOutputValue = "Gamemode is now normal";
+    return myOutputValue;
+  }
+  //Accounting for Draw scenario
+  if (inputSign == resultOfComputerSign) {
+    //calling special message function
+    var specialMessageHere = specialMessage();
+    //calling gamemode message function
+    var gameModeMessageHere = gameModeMessage();
+    var myOutputValue =
```

Comment:
You have declared (var) myOutputValue, gameModeMessageHere, specialMessageHere multiple times in the code. Instead you can consider defining them as global variables and reassigning them inside the various if statements and functions. Remember we only declare a variable once in a code file, but can reassign the value as many times as possible